### ISSUE TITLE
🤖 Train AutoML models for PCE prediction

### ISSUE THREAD

**Issue by newtontech** [newtontech/perovskite-analysis#2]
## Problem

需要训练高性能的机器学习模型来预测钙钛矿太阳能电池的 PCE 效率，实现 R² > 0.85 的目标。

### 1. 数据预处理
```python
# 特征工程
features = [
    'perovskite_band_gap',
    'jv_reverse_scan_j_sc',
    'jv_reverse_scan_v_oc',
    'jv_reverse_scan_ff',
    'cell_area_measured'
]
```

### 2. 模型选择
- [ ] Random Forest Regressor
- [ ] Gradient Boosting Regressor
- [ ] XGBoost Regressor
- [ ] LightGBM Regressor

### 3. 超参数优化
- [ ] Grid Search
- [ ] Random Search
- [ ] Bayesian Optimization

### 4. 模型评估
- [ ] R² Score > 0.85
- [ ] MAE < 1.5%
- [ ] RMSE < 2.0%
- [ ] 交叉验证

### 5. 模型解释
- [ ] 特征重要性
- [ ] SHAP 值分析
- [ ] 预测 vs 实际对比

## Expected Output

- [ ] 最佳模型（R² > 0.85）
- [ ] 特征重要性排序
- [ ] 模型性能报告
- [ ] 预测示例

## Priority

🟢 High - 核心预测功能

🦞 OpenClaw AI Assistant
